apps/journeys-admin/src/components/Editor/ImageBlockEditor/ImageBlockEditor.tsx

```diff
@@ -1,17 +1,21 @@
 import BrushRounded from '@mui/icons-material/BrushRounded'
 import DashboardRounded from '@mui/icons-material/DashboardRounded'
+import SmartToyOutlinedIcon from '@mui/icons-material/SmartToyOutlined'
 import Box from '@mui/material/Box'
 import Tab from '@mui/material/Tab'
 import Tabs from '@mui/material/Tabs'
 import Typography from '@mui/material/Typography'
 import { ReactElement, SyntheticEvent, useState } from 'react'
 import { object, string } from 'yup'
 
+// TODO: remove generative ai flags when ready
+import { useFlags } from '@core/shared/ui/FlagsProvider'
 import { TabPanel, tabA11yProps } from '@core/shared/ui/TabPanel'
 
 import { GetJourney_journey_blocks_ImageBlock as ImageBlock } from '../../../../__generated__/GetJourney'
 import { ImageBlockHeader } from '../ImageBlockHeader'
 
+import { AIGallery } from './AIGallery/AIGallery'
```

Comment:
```suggestion
import { AIGallery } from './AIGallery'
```
add an index file in this folder